pkg/util/workloads_utils.go

```diff
@@ -153,7 +153,7 @@ func EqualIgnoreHash(template1, template2 *v1.PodTemplateSpec) bool {
 }
 
 func HashReleasePlanBatches(releasePlan *v1alpha1.ReleasePlan) string {
-	by, _ := json.Marshal(releasePlan.Batches)
+	by, _ := json.Marshal(releasePlan)
```

Comment:
plz update comments in the. releaseplan.status.ObservedReleasePlanHash

Reply:
done